tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -535,7 +535,7 @@ def check_args(kwargs):
 
     if kwargs['extra_prefs']:
         # If a single pref is passed in as a string, make it a list
-        if type(kwargs['extra_prefs']) in (str, unicode):
+        if isinstance(kwargs['extra_prefs'], string_types+(text_type,)):
```

Comment:
Let's just delete this whole if condition. I don't think we should allow string-or-list as a type here.